semgrep_output_v1.atd

```diff
@@ -686,6 +703,9 @@ type cli_output_extra <ocaml attr="deriving show"> = {
     (* EXPERIMENTAL: since: 1.11.0 *)
     ?rules_by_engine: rule_id_and_engine_kind list option;
     ?engine_requested: engine_kind option;
+
+    (* EXPERIMENTAL: since: 1.37.0 *)
+    ~skipped_rules: skipped_rule list;
```

Comment:
I'm still confused about ~. I understand ?, but each time I see ~ I forget what it does.

Reply:
`~` means a default value is used if the field is missing. See https://atd.readthedocs.io/en/latest/atdpy.html#defining-default-field-values

Reply:
so when you output the JSON, if skipped_rules is the empty list, will it output a skipped_rules: [] in the JSON?
and when you read the JSON, if there is no skipped_rules in the JSON, it will generate a record with an empty list right? (avoiding some annoying match option)

Reply:
I think it would be nice in the doc to give the 2 scenarios, when producing JSON, and when reading JSON, because this has implication on the smooth upgrade process.